bears/vcs/git/GitCommitBear.py

```diff
@@ -203,3 +223,80 @@ def check_body(self, body,
             yield Result(self, 'Body of HEAD commit contains too long lines. '
                                'Commit body lines should not exceed {} '
                                'characters.'.format(body_line_length))
+
+    def check_issue(self, body,
+                    body_regex: str=None,
+                    body_close_issue: bool=False,
+                    body_close_issue_url: bool=False,
+                    body_close_issue_last_line: bool=False):
+        """
+        Check for matching issue related URLs.
+
+        :param body:
+            Body of the commit message of HEAD.
+        :param body_regex:
```

Comment:
So this setting checks the body with a regex, right? Imo this belongs to `check_body`. Also I think if you enable it, it doesn't have to take precedence. For example If I just want that a body message starts with some kind of annotation, like this:
```
JIRA TICKET: [FLINK-1324]

blablablabla body message blabla
```
and my regex would be this one: `JIRA TICKET: \[FLINK-\d+](\n\n.*)?`

Maybe I still want to enforce the URL checks, as my regex is not specific enough + I don't want to reimplement that what you do with this PR inside my regex.